packages/amplify-category-api/src/graphql-transformer/transform-graphql-schema-v2.ts

```diff
@@ -43,6 +43,8 @@ const SCHEMA_FILENAME = 'schema.graphql';
 const SCHEMA_DIR_NAME = 'schema';
 const PROVIDER_NAME = 'awscloudformation';
 const LAYER_MAPPING_URL = 'https://amplify-rds-layer-resources.s3.amazonaws.com/rds-layer-mapping.json';
+const LAYER_MAPPING_URL_BETA = 'https://amplify-rds-layer-resources-beta.s3.amazonaws.com/rds-layer-mapping.json';
```

Comment:
That isn't accurate. Gen1 CLI is still using this layer mapping file. The regionalized S3 layer version files are currently used by the CDK construct only.

I tried to use the custom resource in Gen 1 CLI but the custom resource is looking for S3 deployment bucket and root key as parameters in child stack instead of using the root stack parameters.

In long term, we need to migrate all our E2E tests to use the CDK construct.

Reply:
Updated the PR to use the regionalized S3 manifests. Now Gen1 and CDK construct uses the regionalized S3 manifests.